src/components/form/FluroContentFormField.vue

```diff
@@ -2178,20 +2175,65 @@ export default {
 		dateStringModel: {
 			get() {
 				if (this.timezone) {
-					return this.fieldModel && this.$fluro.date.moment.tz(this.fieldModel, this.timezone).format('YYYY-MM-DD');
+					// console.log("watch timezone field model", this.fieldModel)
+
```

Comment:
If you're just changing a UTC to local, couldn't you just do `moment.utc(this.fieldModel).format('YYYY-MM-DD')`? Or, if you're using the remote timezone, `moment.utc(this.fieldModel).tz(this.timezone).format('YYYY-MM-DD')`?

Reply:
Assuming that "fieldModel" is the time... doesn't sound like it from the variable name :)